firmware/gameport-adapter/Logitech.h

```diff
@@ -63,7 +63,7 @@ class Logitech : public Joystick {
     // read too fast. Following delay will ensure, that after the
     // last read at least 5ms passed to ensure, that the joystick
     // cooled down again.
-    delay(5);
+    delayMicroseconds(5000);
```

Comment:
There is IMHO no real point for this change. If the delay goes into milliseconds and we are not in a time critical code, where the interrupts are deactivated, we easily can use `delay(...)`. But may be I just missed the point, which benefit do you expect from this change?

Reply:
LOL :) Ok, you convinced me